llabake/Courses-Management-PS
# Ft initial app structure

#### What does this PR do?
* Add application page layout

#### Description of Task to be completed?
* Add course page
* Add manage course page
* Add route file to navigate the app
#### How should this be manually tested?
* Clone the repo 
* Install dependencies using `npm install`
* Run `npm start` to start the application
* You should see the browser page open on `localhost:3000`
* Add a course and you should be redirected to the courses pages
* Edit a course and you should be redirected to the courses pages

#### What are the relevant pivotal tracker stories?
* N/A
